### ISSUE TITLE
Title option for Save and Open dialog should mention macOS limitations

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#101443]
Testing #101279

Currently we show this:

![image](https://user-images.githubusercontent.com/35271042/86163127-a7cfef00-bac4-11ea-9535-e906acc59e53.png)

I think we should be explicit here and mention that this is not supported on macOS. So maybe something like:

> Depending on the underlying operating system this parameter might be ignored, since some systems do not present title on save dialogs **(like macOS)**.

**Comment by gregvanl** [comment]
How about rewording:

"This parameter might be ignored, as not all operating systems display a title on open dialogs (for example, macOS)."

Similar for the save dialog text.
